### ISSUE TITLE
A faulty bionics must never be able to occupy more space than the bionic that triggered the faulty installation

### ISSUE THREAD

**Issue by Coolthulhu** [CleverRaven/Cataclysm-DDA#20629]
A blocker to the bionic slot system mainlining.

A faulty installation of a bionic can either occupy 0 slots, exactly the same slots as the actual bionic, less slots and exact same parts as the actual bionics or be tied to the actual bionic.
The situation where player tries to install a bionic that is within limits but ends up with a faulty bionic above it must never happen.

Honestly the last option sounds the best here: it's the bionic which is faulty, not a fault which is a bionic. Removal should require getting rid of the properly working bionic too.

**Comment by DangerNoodle** [comment]
This is why I recommended faulty bionics only have the minimum of 1 slot size, if they use any at all. Simplest option, as there is little gained from them having any size at all.

**Comment by kevingranade** [comment]
I agree faults being part of the (potentially still functional) bionic
makes the most sense, including requiring removal of the host bionic to
remove the faults.

**Comment by fengor** [comment]
Depends. A big enough failure should be able to make the bionic totally non functionable.

Something like this for example
Success: functional bionic
Failure: functional but with a faulty "attachment" (maybe make the attachment 0Size)
Huge Failure: replace the functional bionic with a faulty bionic with size >0
